src/core/scene/a-scene.js

```diff
@@ -157,8 +159,10 @@ module.exports = registerElement('a-scene', {
           self.resize();
           self.emit('exit-vr', {target: self});
         }
-        function exitVRFailure () {
-          throw new Error('exit VR mode error. exitPresent failed');
+        function exitVRFailure (err) {
+          var reason = err && err.message;
+          throw new Error('Failed to exit VR mode (`exitPresent`).' +
+            (reason ? '\n' + reason : ''));
```

Comment:
can you move this to a variable?
`var reasonString = reason ? '\n' + reason : '';`